orttraining/orttraining/training_ops/cuda/optimizer/lamb.cc

```diff
@@ -542,9 +544,19 @@ Status LambOptimizer<T1, T2, T3, T4, T_GRAD_NORM, T_MIXED_PRECISION_FP>::Compute
     max_tensor_size = std::max(max_tensor_size, static_cast<int>(w.Shape().Size()));
   }
 
-  // Allocate a buffer in byte for reduction API calls.
-  const auto reduction_buffer_size =
-      compute_reduction_buffer_size<CudaT2>(max_tensor_size);
+  const size_t reduction_buffer_size = [&](){
+
+    // Allocate a buffer in byte for reduction API calls.
+    size_t rbs = compute_reduction_buffer_size<CudaT2>(max_tensor_size);
+
+    // Enlarge reduction buffer to accomodate multi-tensor reduction kernel as well
+    const int tensor_group_size = 4; // w, d, w_norm, d_norm
+    const int max_blocks = ChunkGroup<tensor_group_size>::max_block_count;
+    const size_t multitensor_block_reduce_buffer_size = 2*max_blocks*sizeof(CudaT2);
```

Comment:
>* [](start = 57, length = 1)

Add spaces between operands? Maybe format this file?

Reply:
Prefer not to change other parts of file, so just added spacing about operands.